src/colmap/estimators/covariance.h

```diff
@@ -115,6 +115,10 @@ std::optional<BACovariance> EstimateBACovariance(
     const BACovarianceOptions& options,
     const Reconstruction& reconstruction,
     BundleAdjuster& bundle_adjuster);
+std::optional<BACovariance> EstimateBACovariance(
```

Comment:
Let's also name this EstimateBACovarianceFromProblem ?

Reply:
Thanks. This is done.